test/integration/authorisation_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class AuthorisationTest < ActionDispatch::IntegrationTest
```

Comment:
`Authorisation` and `Authentication` stand for different purposes. This test is about `Authentication`.